### ISSUE TITLE
possibility to set a global resfinder species value for samples in YAML

### ISSUE THREAD

**Issue by fmalmeida** [fmalmeida/bacannot#33]
Add the possibility to use the `--resfinder_species` parameter outside the YAML file to set a global value for all samples described in the YAML. Which can then be overwritten if a specific sample has another value for it in samplesheet.

It will work like this, for example:

`--resfinder_species Klebsiella` will set all samples from the YAML as Klebsiella.

However, if the samplesheet is like this:

```yaml
samplesheet:
  - id: sample_1
    illumina:
      - sample_1/1.fastq
      - sample_1/2.fastq
    nanopore: sample_1/ont.fastq
    resfinder: Escherichia coli
  - id: sample_2
    assembly: sample_2/assembly.fasta
  - id: sample_3
    nanopore: sample_3/ont.fastq
    fast5: sample_3/fast5_pass
```

It will run sample 2 and 3 as the global value "Klebsiella" and sample 1 will be run as "Escherichia coli" since it overwrites the global definition by setting a value inside the YAML.

**Comment by fmalmeida** [comment]
This is now being tackled in branch https://github.com/fmalmeida/bacannot/tree/develop together with issue https://github.com/fmalmeida/bacannot/issues/31

**Comment by fmalmeida** [comment]
Solved in new version 3.0.
